### ISSUE TITLE
Documentation update.

### ISSUE THREAD

**Issue by poojashresthacode** [ubsuny/23-Homework4G4#70]
Hi @WildJimmy ,
I have made the following changes to the unit test code.

- I have created a separate module for the unit test.
- trailing spaces were minimized.
- Input was given manually in the code.
- actual_periodic_times was changed to actual_p_t to make the line shorter.

Can you please add this to the documentation file?

Also, add the references too.

**Comment by poojashresthacode** [comment]
I mean to say, you can enlist or paste the following mentioned things in the documentation file.

**Comment by WildJimmy** [comment]
Yep, I'll make an effort to include this in the documentation before the deadline
